### ISSUE TITLE
Test usefullness of pre-training

### ISSUE THREAD

**Issue by Etienne-bdt** [Etienne-bdt/Simple-VAE-RS#9]
Now that everything works well, is pre-training still usefull ?

**Comment by Etienne-bdt** [comment]
Pre-training seems to improve loss in the end, maybe because pre-training the HR recon without constraint first improves the HR pipeline in such a way that the KLD for **z** conditioned on **u** and **y** will not hinder the reconstruction as much as without pre-training. Testing with more pre-epochs might help us find out if converging first without the conditional pipeline can improve convergence speed with the conditional on

**Comment by Etienne-bdt** [comment]
After further investigation, it has been observed that when pre-training converges, training reaches different local minimas. As of May 9th, we could see that reconstruction capacity are about similar for both trainings (with and without 100 epochs of pre-training). The dataset is now used at its full extent (grid cropping yielding 16 patches par image instead of 1 random) meaning that each epoch the model sees 16* data. This might have helped convergence of the not pre-trained model.
